src/components/transport_manager/src/usb/libusb/usb_connection.cc

```diff
@@ -192,43 +194,50 @@ bool UsbConnection::PostOutTransfer() {
   if (LIBUSB_SUCCESS != libusb_ret) {
     LOG4CXX_ERROR(
         logger_,
-        "libusb_submit_transfer failed: " << libusb_error_name(libusb_ret)
-                                          << ". Abort connection.");
-    AbortConnection();
+        "libusb_submit_transfer failed: " << libusb_error_name(libusb_ret));
     LOG4CXX_TRACE(logger_,
                   "exit with FALSE. Condition: "
                       << "LIBUSB_SUCCESS != libusb_fill_bulk_transfer");
-    return false;
+    return TransportAdapter::FAIL;
   }
   LOG4CXX_TRACE(logger_, "exit with TRUE");
-  return true;
+  return TransportAdapter::OK;
 }
 
 void UsbConnection::OnOutTransfer(libusb_transfer* transfer) {
   LOG4CXX_TRACE(logger_, "enter with  Libusb_transfer*: " << transfer);
-  sync_primitives::AutoLock locker(out_messages_mutex_);
-  if (transfer->status == LIBUSB_TRANSFER_COMPLETED) {
-    bytes_sent_ += transfer->actual_length;
-    if (bytes_sent_ == current_out_message_->data_size()) {
-      LOG4CXX_DEBUG(
+  auto error_code = TransportAdapter::OK;
+  {
+    sync_primitives::AutoLock locker(out_messages_mutex_);
+    if (transfer->status == LIBUSB_TRANSFER_COMPLETED) {
+      bytes_sent_ += transfer->actual_length;
+      if (bytes_sent_ == current_out_message_->data_size()) {
```

Comment:
@SNiukalov same as above

Reply:
@AByzhynar 
Thanks, fixed in:https://github.com/smartdevicelink/sdl_core/pull/3332/commits/0a50e19ee0dc5655e45a277c67cf9d42c8fc16bd